src/main/java/mate/controller/CarCreationController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+public class CarCreationController extends HttpServlet {
```

Comment:
Fix it everywhere 
<img width="915" alt="Screen Shot 2022-10-18 at 7 47 26 PM" src="https://user-images.githubusercontent.com/90692308/196493189-76fd00f2-3d74-473d-84c2-8e5f8d83ebdc.png">